.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@n3m0-22 make sure your editor settings don't get committed

Reply:
Sorry about that. I'll clean it up.